source/server-discovery-and-monitoring/tests/integration/closeInUse-pool-clear.yml

```diff
@@ -0,0 +1,108 @@
+# Test connection pool clear(closeInUse:true).
+runOn:
+    # This test requires failCommand appName, and
+    # failCommand blockConnection with closeConnection:true (SERVER-53512).
+    - minServerVersion: "4.9"
+
+database_name: &database_name "sdam-tests"
+collection_name: &collection_name "closeInUse"
+
+data: []
+
+tests:
+  - description: pool clear close in use true generates expected events
+    useMultipleMongoses: false,
+    clientOptions:
+      connectTimeoutMS: 500, 
+      heartbeatFrequencyMS: 500
+      appname: "closeInUse"
+      retryReads: false
+      minPoolSize: 0
+    operations:
+      - name: insertMany
+        object: collection
+        arguments:
+          documents:
+            - _id: 1
+      - name: startThread
+        object: testRunner
+        arguments:
+          name: thread1
+      # simulate a long-running query
+      - name: runOnThread
+        object: testRunner
+        arguments:
+          name: thread1
+          operation:
+            name: find
+            object: collection
+            arguments:
+              filter: { "$where": "sleep(5000) || true" }
+      - name: configureFailPoint
+        object: testRunner
+        arguments:
+          failPoint:
+            configureFailPoint: failCommand
+            mode: { times: 1 }
+            data:
+                failCommands: ["hello"]
+                blockConnection: true
+                blockTimeMS: 1000
+                appName: closeInUse
+      - name: waitForEvent
+        object: testRunner
+        arguments:
+          event: ConnectionClosed
+          count: 1
+      - name: waitForEvent
+        object: testRunner
+        arguments:
+          event: PoolClearedEvent
```

Comment:
can we add a `closeInUseConnections: true` assertion here? We'll need to update the [Unified Format schema](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/schema-1.10.json#L432) and bump the version as part of that. Be sure to update the changelog as well.

Reply:
done